docs/configuration/metrics/index.md

```diff
@@ -41,6 +41,7 @@ Every metric that is being declared needs to define the following fields:
 
 # Supported Azure Services
 
+- [Generic Azure Resource](generic-azure-resource)
```

Comment:
Maybe some guidance on when one would use the Generic Azure Resource vs a more specific one? Ease of use?

Reply:
Limited support for more convenient ones but good point. Will update this